packages/office-ui-fabric-react/src/components/Announced/examples/Announced.BulkOperations.Example.tsx

```diff
@@ -17,44 +16,17 @@ const dragEnterClass = mergeStyles({
   backgroundColor: theme.palette.neutralLight
 });
 
-const _columns: IColumn[] = [
-  {
-    key: 'name',
-    name: 'Name',
-    fieldName: 'name',
+const _columns: IColumn[] = ['Name', 'Modified', 'Modified By', 'File Size'].map((name: string) => {
+  const fieldName = name.replace(' ', '').toLowerCase();
+  return {
+    fieldName,
+    name,
+    key: fieldName,
     minWidth: 100,
     maxWidth: 200,
-    isResizable: true,
-    ariaLabel: 'Operations for name'
```

Comment:
Do we not need these `ariaLabel` anymore? Would these make Keros scans complain about this example page?

Reply:
In this case the column has a visible name that the screen reader can read, and the aria label was actually wrong because this list doesn't support column operations. It's better to just read the visible text than to have a more detailed but incorrect aria label.